web/package.json

```diff
@@ -181,6 +181,8 @@
     }
   },
   "prettier": {
-    "singleQuote": true
+    "singleQuote": true,
+    "trailingComma": "none",
+    "arrowParens": "avoid"
```

Comment:
With prettier options stated here, cli flags for the `npm run prettier` command are likely unnecessary.

Reply:
yeah I was thinking that too. I'm going to move this config to a file at the top folder level and update the command